server/api/excused_absence/excused_absence.controller.js

```diff
@@ -115,9 +122,13 @@ exports.update = (req, res) => {
 * @apiError (500) UnknownException Could not destroy ExcusedAbsence model
 */
 exports.destroy = (req, res, next) => {
-  // TODO - ensure this is only deletable by the user who created the record, or an admin
-  return ExcusedAbsence.remove({ _id: req.params.id })
-  .then((response) => {
-      return res.status(200).send(response).end()
-  }).catch(next)
-}
+    
+  if(auth.hasRole('admin') || (auth.isAuthenticated === ExcusedAbsence.findById(req.params.id).user )) {
+      return ExcusedAbsence.remove({ _id: req.params.id })
+      .then((response) => {
+          return res.status(200).send(response).end()
+      }).catch(next)}
+  
+  else {
+    return res.status(401).send(response).end()
+  }
```

Comment:
Ooops! You're missing a `}` here to close off the `exports.destroy = (...) => {` statements

Just add a last `}` at line `135` and push that change

Reply:
fixed that and pushed the changes!